sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/models/_conversation_analysis_client_enums.py

```diff
@@ -30,6 +80,31 @@ class ErrorCode(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
     INTERNAL_SERVER_ERROR = "InternalServerError"
     SERVICE_UNAVAILABLE = "ServiceUnavailable"
 
+class ExtraInformationKind(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
```

Comment:
This looks really suspect.... do we know where/how this is used?

Reply:
"Extra info" is supposed to be part of the new features added to this release
Farhad from the service team said they're still working on it
I'll ping him in order to update our samples

Reply:
i added the samples